contracts/facets/EmbalmerFacet.sol

```diff
@@ -236,32 +240,34 @@ contract EmbalmerFacet {
         }
 
         // Confirm resurrection time has not yet passed
-        if (sarcophagus.resurrectionTime <= block.timestamp) {
-            revert LibErrors.SarcophagusIsUnwrappable();
+        if (block.timestamp >= sarcophagus.resurrectionTime) {
+            revert LibErrors.ResurrectionTimeInPast(sarcophagus.resurrectionTime);
         }
 
         // Confirm that new resurrection time is in future
-        if (resurrectionTime <= block.timestamp) {
+        if (block.timestamp >= resurrectionTime) {
             revert LibErrors.NewResurrectionTimeInPast(resurrectionTime);
         }
 
         // Confirm that new resurrection time doesn't exceed sarcophagus's maximumRewrapInterval
-        if (resurrectionTime > block.timestamp + sarcophagus.maximumRewrapInterval) {
+        if (block.timestamp + sarcophagus.maximumRewrapInterval < resurrectionTime) {
```

Comment:
Is there a reason you seem to be making sure `block.timestamp` appears earlier in these expressions? Some efficiency/edgecase shenanigans maybe? Or just a style choice?

Reply:
it is just a style thing haha I was running through all the code trying to evaluate it for readability and I personally think it's easier to follow if block timestamp is consistently the first thing in the checks